app/backend/lib/s3admin-archive.ts

```diff
@@ -0,0 +1,60 @@
+import { Router } from 'express';
+
+import config from '../../config';
+import getAuthRole from '../../utils/getAuthRole';
+import {checkFileExists,getFileFromS3} from './s3client';
+import {pushMessage} from './sns-client';
+import getAttachementList from './attachements';
+import getLastIntakeId from './lastIntake';
+
+const AWS_S3_BUCKET = config.get('AWS_S3_BUCKET');
+const AWS_S3_SECRET_KEY = config.get('AWS_S3_SECRET_KEY');
+const ARCHIVE_REQUEST_TOPIC_ARN = config.get('ARCHIVE_REQUEST_TOPIC_ARN');
+
+const s3adminArchive = Router();
+
+// eslint-disable-next-line consistent-return
+s3adminArchive.get('/api/analyst/admin-archive/:intake', async (req, res) => {
+  const authRole = getAuthRole(req);
+  const isRoleAuthorized =
+    authRole?.pgRole === 'ccbc_admin' || authRole?.pgRole === 'ccbc_analyst';
+  if (!isRoleAuthorized) {
+    // check header
+    const apiKey = req.headers['x-api-key'];
+    if (apiKey !== AWS_S3_SECRET_KEY) {
+      return res.status(404).end();
+    }
+    else {
+      req.claims= {
+        identity_provider:'idir',
+        client_roles: ['admin'],
+        aud:'ccbc', exp:0, iat:0, iss:'ccbc', sub:'ccbc'
+      } 
```

Comment:
why are we manually setting req.claims here?

Reply:
Because graphql.performQuery uses psSettings which are set by authenticationPgSettings(Request) where claims are needed.

Reply:
Nice, thanks I was just curious